### PR TITLE
Create http4s Laika/Helium theme

### PR DESCRIPTION
**Pull request by armanbilge**
Closes https://github.com/http4s/sbt-http4s-org/issues/140.

Preview locally with `docs/tlSitePreview` but it's pretty much the same as what we already have.

### CODE REVIEW

File: core/src/main/resources/org/http4s/sbt/site/default.template.html [hunk 1]
```diff
@@ -29,20 +29,21 @@
...
           <a id="version-menu-toggle" class="text-link drop-down-toggle" href="#">
-            Version 1.0
+            ${helium.topBar.versionPrefix} ${_.currentVersion.displayValue}
           </a>
           <nav id="version-menu">
             <ul id="version-list" class="nav-list">
             </ul>
           </nav>
         </div>
+        @:@
```
Comment (armanbilge) [inline core/src/main/resources/org/http4s/sbt/site/default.template.html:46]: This will enable versioned sites 👍

**Review by rossabaker — APPROVED** [review]
Looks good

**Comment by rossabaker** [comment]
Is this ready for a release then?

**Comment by armanbilge** [comment]
Depends what you think of https://github.com/http4s/http4s-jdk-http-client/pull/582?

**Pull request merged**